eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -48,16 +48,31 @@ jobs:
 
       - template: ../steps/common.yml
 
-      - script: |
-          node common/scripts/install-run-rush.js install
-        displayName: "Install dependencies"
-
       - pwsh: |
           $folder = "${{parameters.ServiceDirectory}}"
           if ($folder -eq "*") { $folder = "" }
           echo "##vso[task.setvariable variable=folder]$folder"
         displayName: "Set folder variable for readme links"
 
+      - pwsh: |
+          $setDailyDevBuild = "false"
+          if (('$(Build.Reason)' -eq 'Schedule') -and ('$(System.TeamProject)' -eq 'internal')) {
+            $setDailyDevBuild = "true"
+          }
+          echo "##vso[task.setvariable variable=DevBuildRun]$setDailyDevBuild"
+        displayName: "Setup Versioning Properties"
+
+      - script: |
+          node eng/tools/versioning/set-dev.js --build-id "$(Build.BuildNumber)" --repo-root "$(Build.SourcesDirectory)" --service "$(folder)"
```

Comment:
I think you need to npm install the dependenices for set-dev before you call it otherwise the semver dependencies might be missing. Correct @danieljurek?

Reply:
Correct. The package requires `semver` and `yargs` to work properly

Reply:
added install step